dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp

```diff
@@ -1,74 +1,160 @@
+                    <button
+                        dojoType="dijit.form.Button"
+                        dojoAttachEvent="onClick:_doRelateContent"
+                        iconClass="searchIcon"
+                    >
+                        <%= LanguageUtil.get(pageContext, "Relate") %>
+                    </button>
+                </div>
+            </div>
+            <table
+                dojoAttachPoint="results_table"
+                class="listingTable relateContent"
+            ></table>
+        </div>
+        <div class="portlet-pagination">
+            <div dojoAttachPoint="previousDiv" style="display: none;">
+                <button
+                    dojoType="dijit.form.Button"
+                    class="bg"
+                    dojoAttachEvent="onClick:_previousPage"
+                    iconClass="previousIcon"
+                >
+                    <%= LanguageUtil.get(pageContext, "Previous") %>
+                </button>
+            </div>
+            <div></div>
+            <div dojoAttachPoint="nextDiv" style="display: none;">
+                <button
+                    dojoType="dijit.form.Button"
+                    class="bg"
+                    dojoAttachEvent="onClick:_nextPage"
+                    iconClass="nextIcon"
+                >
+                    <%= LanguageUtil.get(pageContext, "Next") %>
+                </button>
+            </div>
+        </div>
+    </div>
+</form>
 
 <!-- Dynamic Variables/Language specific content from request/session/context -->
 <div style="display: none;">
-    <input type="hidden" dojoAttachPoint="htmlPageLanguage" value="<%=session.getAttribute(com.dotmarketing.util.WebKeys.HTMLPAGE_LANGUAGE) %>">
-    <input type="hidden" dojoAttachPoint="tagText" value="<%= LanguageUtil.get(pageContext, "Type-your-tag-You-can-enter-multiple-comma-separated-tags") %>">
-    <input type="hidden" dojoAttachPoint="suggestedTagsText" value="<%= LanguageUtil.get(pageContext, "Suggested-Tags") %>">
-    <input type="hidden" dojoAttachPoint="noResultsText" value="<%= LanguageUtil.get(pageContext, "No-Results-Found") %>">
-    <input type="hidden" dojoAttachPoint="matchResultsText" value="<%= LanguageUtil.get(pageContext, "Matching-Results") %>">
+    <input
+        type="hidden"
+        dojoAttachPoint="htmlPageLanguage"
+        value="<%=session.getAttribute(com.dotmarketing.util.WebKeys.HTMLPAGE_LANGUAGE) %>"
+    />
+    <input type="hidden" dojoAttachPoint="tagText" value="<%=
+    LanguageUtil.get(pageContext,
+    "Type-your-tag-You-can-enter-multiple-comma-separated-tags") %>"> <input
+    type="hidden" dojoAttachPoint="suggestedTagsText" value="<%=
+    LanguageUtil.get(pageContext, "Suggested-Tags") %>"> <input type="hidden"
+    dojoAttachPoint="noResultsText" value="<%= LanguageUtil.get(pageContext,
+    "No-Results-Found") %>"> <input type="hidden"
+    dojoAttachPoint="matchResultsText" value="<%= LanguageUtil.get(pageContext,
+    "Matching-Results") %>">

```

Comment:
it is hard to see that we have multi inputs hidden tag here, maybe you should split them into several lines

Reply:
+1